kernel/.merlin.in

```diff
@@ -0,0 +1,8 @@
+FLG -rectypes -thread -safe-string -w +a-4-44-50
+
+S ../clib
+B ../clib
+S ../config
+B ../config
+S ../lib
+B ../lib
```

Comment:
Can we just use
~~~
REC
FLG -w +a-4-44-50
~~~
?
Or does that not work?

Reply:
That does not work, at least in my setup.